src/Effects.js

```diff
@@ -670,7 +655,10 @@ module.exports = class Effects {
           const poop = this.poop[i];
           p5.translate(poop.x, poop.y);
           p5.rotate(poop.rot);
+          p5.push();
+          p5.scale(2, 2);
           drawFallingPoop(p5._renderer.drawingContext);
+          p5.pop();
```

Comment:
This one is balanced, but I don't think we need this `push/pop` at all - no drawing operations occur between this `pop()` and the next one, suggesting they're redundant.

Reply:
will address these throughout. thanks!